scripts/xe-reset-networking

```diff
@@ -136,8 +137,11 @@ if __name__ == "__main__":
 		if vlan != None:
 			configuration.append("Vlan:                   " + vlan)
 		if options.mode == "static":
-			configuration.append("IP address:             " + options.ip)
-			configuration.append("Netmask:                " + options.netmask)
+			if not options.use_ipv6:
+				configuration.append("IP address:             " + options.ip)
+				configuration.append("Netmask:                " + options.netmask)
+			else:
+				configuration.append("IPv6 address:           " + options.ip + "/" + options.netmask)
```

Comment:
...and `--ipv6-gateway`.

Reply:
sure, so I should add a specific gateway_v6 and dns_v6 as well right?

Reply:
Well right now, the list of DNS servers is common to v4 and v6. Also the mode has different options for IPv6. So perhaps have `--ipv6`, `--mode-v6`, `--gateway-v6` (use `-` rather than `_` for the CLI).

Reply:
Okay, i'll use specific field!